dev/sparktestsupport/modules.py

```diff
@@ -165,11 +176,32 @@ def __hash__(self):
     build_profile_flags=[
         "-Phive-thriftserver",
     ],
+    environ={
+        "SUPPORTED_HADOOP_PROFILE": "hadoop2.7"
+    },
     sbt_test_goals=[
         "hive-thriftserver/test",
     ]
 )
 
+hive_thriftserverV2 = Module(
+    name="hive-thriftserverV2",
+    dependencies=[hive],
+    source_file_regexes=[
+        "sql/hive-thriftserverV2",
+        "sbin/start-thriftserver.sh",
+    ],
+    build_profile_flags=[
+        "-Phive-thriftserverV2",
+    ],
+    environ={
+        "SUPPORTED_HADOOP_PROFILE": "hadoop3.2"
```

Comment:
can hive/thriftserver 2 run on hadoop 2.7?

Reply:
Yes. It can run on both hadoop 2.7 and hadoop 3.2. But it can't run on Hive 1.2.1. We did not add more profiles for Hive 1.2.1 and Hive 2.3.4. So use the Hadoop profile to distinguish it.
How to build it for hadoop 2.7:
```shell
./dev/make-distribution.sh --name hadoop-2.7_hive-2.3 --tgz -Phadoop-3.2 -Phive -Phive-thriftserverV2 -Pyarn -Dhadoop.version=2.7.4 -Dzookeeper.version=3.4.6 -Dcurator.version=2.7.1
```